### PR TITLE
added product Schemas

### PR DESCRIPTION
**Pull request by prabhum2**
Please link to the issue #…

### CODE REVIEW

File: components/classes/product.schema.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          "description": "Name of the product."
+        },
+        "xdm:productDescription": {
+          "title": "Product Description",
+          "type": "string",
+          "description": "Description of the product."
+        },
+        "xdm:productRating": {
+          "title": "Product Rating",
+          "type": "string",
```
Comment (mjasrotia) [inline components/classes/product.schema.json:37]: Should this be a number ?

File: components/classes/product.schema.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          "description": "Customer review rating of the product."
+        },
+        "xdm:productListPrice": {
+          "title": "Product List price",
+          "type": "number",
```
Comment (mjasrotia) [inline components/classes/product.schema.json:42]: should we use the currency type here ?

File: components/mixins/product/product-catelog-category.schema.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/product-catelog-category",
```
Comment (mjasrotia) [inline components/mixins/product/product-catelog-category.schema.json:8]: catalog

File: components/mixins/product/product-catelog.schema.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/product-catelog",
```
Comment (mjasrotia) [inline components/mixins/product/product-catelog.schema.json:8]: catalog

File: components/mixins/product/product-identifiers.schema.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  "type": "object",
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
+  "description": "This mixin is used to capture properties related to product identifiers generic to all industries.",
+  "definitions": {
+    "product-identifiers": {
+      "properties": {
+        "xdm:productSku": {
```
Comment (mjasrotia) [inline components/mixins/product/product-identifiers.schema.json:20]: should this be productSKU ?

File: components/mixins/product/product-identifiers.schema.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+          "title": "Product SKU",
+          "type": "string",
+          "description": "The unique SKU (stock keeping unit) of the product assigned by the vendor."
+        },
+        "xdm:productGtin": {
```
Comment (mjasrotia) [inline components/mixins/product/product-identifiers.schema.json:25]: productGTIN ?

File: components/mixins/product/product-identifiers.schema.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+          "description": "The unique SKU (stock keeping unit) of the product assigned by the vendor."
+        },
+        "xdm:productGTIN": {
+          "title": "Product GTIN",
+          "type": "string",
+          "description": "Global Trade Item Number of the product."
+        },
+        "xdm:productImageURL": {
+          "title": "Product Image URL",
+          "type": "string",
```
Comment (mjasrotia) [inline components/mixins/product/product-identifiers.schema.json:32]: Should this be a "uri" format ?

**Review by mjasrotia — APPROVED** [review]

**Review by cmathis — APPROVED** [review]

**Pull request merged**